_ml-commons-plugin/api.md

```diff
@@ -115,9 +115,191 @@ The API returns information on the model, the algorithm used, and the content fo
 }
 ```
 
+## Load a model
+
+Use the upload operation to upload a custom model and into a model index. ML commons splits the model into smaller chunks and saves those chunks into the model's index.
+
+```json
+POST /_plugins/_ml/models/_upload
+```
+
+### Request fields
+
+Field | Data Type | Description
```

Comment:
Could we also specify which body fields are required and which ones are optional?